pyaedt/edb_core/EDB_Data.py

```diff
@@ -587,6 +589,50 @@ def filling_material_name(self, value):
             self._filling_material_name = value
             self.update_layers()
 
+    @property
+    def negative_layer_value(self):
```

Comment:
```suggestion
    def negative_layer(self):
```
I would rename it as ``negative_layer``

Reply:
Ok i will do it